apollo-router/src/plugins/telemetry/mod.rs

```diff
@@ -300,6 +300,25 @@ impl Plugin for Telemetry {
                                         start.elapsed(),
                                     );
                                 }
+                                let mut metric_attrs = Vec::new();
+                                dbg!(&e);
```

Comment:
Replace this with `tracing::debug` or delete it.

Reply:
Deleted ! Good catch thanks